### ISSUE TITLE
Windows 10: leaving screen idle scans a value!?

### ISSUE THREAD

**Issue by divukman** [phonegap/phonegap-plugin-barcodescanner#559]
### Expected Behaviour

Expecting no value to be scanned.

### Actual Behaviour

Leaving screen idle (not scanning anything) results with a value scanned. Each time different random value.

### Reproduce Scenario (including but not limited to)

Normal scanning option. 

#### Steps to Reproduce

Leave to timeout. 
Put a sticker on a camera, it scans a value at random interval.

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)

Windows 10

#### Plugin version

    phonegap-plugin-barcodescanner 6.0.8 "BarcodeScanner"

Basically, zxingReader.decode returns a result that has a text value even though there should be none.
